snippets/card-product.liquid

```diff
@@ -7,7 +7,8 @@
     - show_secondary_image: {Boolean} Show the secondary image on hover. Default: false (optional)
     - show_vendor: {Boolean} Show the product vendor. Default: false
     - show_rating: {Boolean} Show the product rating. Default: false
-    - extend_height: {Boolean} Card height extends to available container space. Default: false (optional)
+    - extend_height: {Boolean} Card height extends to available container space. Default: true (optional)
+    - lazy_load: {Boolean} Image should be lazy loaded. Default: true (optional)
```

Comment:
Same here if we keep the if approach we would need to add `lazy_load` for `collage.liquid` and a few other places.